src/config/supportIssues.js

```diff
@@ -121,7 +134,7 @@ const augmentIssuesList = function () {
   return _.map(supportIssues, (supportIssue) => ({
     ...supportIssue,
     messageLabel: supportIssue.messageLabel || defaultMessageLabel,
-    condition: supportIssue.condition || idConditionsMap[supportIssue.id] || defaultCondition
+    condition: supportIssue.condition || defaultCondition
```

Comment:
We can remove `hasGrants` from the default condition and individual conditions and just add it at the end. 
```suggestion
    condition: all(supportIssue.condition || defaultCondition, hasGrants('support/manage'))
```
We could also revert every change and do

```suggestion
    condition: all(supportIssue.condition || idConditionsMap[supportIssue.id] || defaultCondition, hasGrants('support/manage'))
```

OR even make changes in `selectors/support` instead of here.